lib/Parser/RdfXml.php

```diff
@@ -183,6 +183,11 @@ protected function splitURI($v)
     /** @ignore */
     protected function add($s, $p, $o, $sType, $oType, $oDatatype = null, $oLang = null)
     {
+        if ($oDatatype !== null) {
+            // Typed literals are not affected by language tags.
+            // See: https://github.com/sweetrdf/easyrdf/issues/76
+            $oLang = null;
+        }
```

Comment:
Isn't this per se setting `$oDatatype` always to `null`, no matter if `$oLang` is given?

If no `$oLang` is given, setting `$oDatatype` should be possible. Or did I overlook something?

In the following a suggestion how I would counter the issue:

```suggestion
        /**
         * In case $oLang and $oDatatype are given, we ignore $oLang.
         * 
         * FYI: https://github.com/sweetrdf/easyrdf/issues/76
         */
        if (null !== $oLang && null !== $oDatatype) {
            $oLang = null;
        }
```